### PR TITLE
Add sqlalchemy

### PR DESCRIPTION
**Pull request by Niccolum**
**Issue:** [closes #5436](https://github.com/simple-icons/simple-icons/issues/5346)

**Similarweb rank:**
https://www.similarweb.com/website/sqlalchemy.org/#overview
Global Rank: 125,029
Country Rank: 153,737
Category Rank: 2,972

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -12558,6 +12558,15 @@
             "hex": "FF0000",
             "source": "https://www.spyder-ide.org"
         },
+        {
+            "title": "SQLAlchemy",
+            "hex": "D71F00",
+            "source": "https://user-images.githubusercontent.com/128223/153606192-c117618d-cabf-49c4-9204-d7f1afe24289.svg",
+            "guidelines": "https://github.com/sqlalchemy/sqlalchemy/discussions/7672#discussioncomment-2157650",
```
Comment (adamrusted) [inline _data/simple-icons.json:12565]:
```suggestion
            "source": "https://github.com/sqlalchemy/sqlalchemy/discussions/7672#discussioncomment-2157650",
```

**Review by adamrusted — APPROVED** [review]
Icon looks spot on to me, I'd remove the direct link as the source URL, and instead use the guidelines link. Will merge this now, and we can get this closed. Thanks @Niccolum! 🎉

File: .jsonschema.json [hunk 2]
```diff
@@ -680,7 +680,7 @@
         "pattern": "^https://github\\.com/(?!(features/actions)|(sponsors)|(logos)$)"
       },
       "then": {
-        "pattern": "^https://github\\.com/[^/]+/[^/]+/(blob/[a-f\\d]{40}/[^\\s]+)|(tree/[a-f\\d]{40}(/[^\\s]+)?)|(((issues)|(pull))/\\d+#issuecomment-\\d+)$"
+        "pattern": "^https://github\\.com/[^/]+/[^/]+/(blob/[a-f\\d]{40}/[^\\s]+)|(tree/[a-f\\d]{40}(/[^\\s]+)?)|(((issues)|(pull)|(discussions))/\\d+#(issuecomment|discussioncomment)-\\d+)$"
```
Comment (adamrusted) [inline .jsonschema.json:683]: @simple-icons/maintainers - can someone confirm the change to the JSON schema is okay for merging? Rest of the PR is fine.

Comment (mondeja) [inline .jsonschema.json:683]: Please, group the words as `(issues)|(pull)|(discussions)`:

```suggestion
        "pattern": "^https://github\\.com/[^/]+/[^/]+/(blob/[a-f\\d]{40}/[^\\s]+)|(tree/[a-f\\d]{40}(/[^\\s]+)?)|(((issues)|(pull)|(discussions))/\\d+#((issuecomment)|(discussioncomment))-\\d+)$"
```

**Review by mondeja — CHANGES_REQUESTED** [review]

**Comment by Niccolum** [comment]
Hello everyone!

@adamrusted i updated @mondeja 's suggestion. Can we merge it now?

**Review by mondeja — APPROVED** [review]
Thanks!

**Pull request merged**